sections/homepage-getting-started.js

```diff
@@ -58,18 +61,33 @@ const HomepageGettingStarted = () => (
       To download styled-components run \`npm install --save styled-components\`.
       That's all you need to do, you are now ready to use it in your app! (yep, no build step needed 👌)
 
+      > It's highly recommended to also use the [styled-components babel plugin](https://github.com/styled-components/babel-plugin-styled-components). It offers many benefits like more legible class names, server-side rendering compatibility, smaller bundles, and more.
```

Comment:
Let's make sure to make it clear that it's not recommended, maybe something like

> If you do have access to the build configuration and are using Babel it's recommended to use the styled-components babel plugin. (it is not required to use styled-components though, unless you're server-side rendering) It offers many benefits like more legible class names, server-side rendering compatibility, smaller bundles, and more.